integration/kubernetes/untrusted_workloads/pod-pv-volume.yaml

```diff
@@ -0,0 +1,17 @@
+kind: PersistentVolume
+apiVersion: v1
+metadata:
+  name: pv-volume
+  annotations:
```

Comment:
these annotations are not needed here as this is a persistent volume, not a pod. 
same comment for the `pod-volume-claim.yaml`